test/src/406-ducc-webhook-notifications/main

```diff
@@ -0,0 +1,97 @@
+cvmfs_test_name="Webhook notifications test"
+cvmfs_test_autofs_on_startup=false
+cvmfs_test_suites="ducc"
```

Comment:
Let's add this to the "quick" test suite as well and kick of the cloud tests again. I think otherwise it doesn't run.